### PR TITLE
Fix undefined variable $tagId

### PR DESCRIPTION
**Pull request by nickvergessen**
@MorrisJobke @LukasReschke

### CODE REVIEW

**Comment by karlitschek** [comment]
makes sense :+1:

File: apps/files/lib/activityhelper.php [hunk 1]
```diff
@@ -47,7 +47,11 @@ public function __construct(ITagManager $tagManager) {
 	 */
 	public function getFavoriteFilePaths($user) {
 		$tags = $this->tagManager->load('files', [], false, $user);
-		$favorites = $tags->getFavorites();
+		try {
+			$favorites = $tags->getFavorites();
+		} catch (\Exception $e) {
+			throw new \RuntimeException('No favorites', 1);
```
Comment (MorrisJobke) [inline apps/files/lib/activityhelper.php]: Can we please not simply swallow any exception. Either add it as the third parameter (previous exception https://php.net/manual/de/exception.construct.php ) or append the message or log the previous message, but please don't drop it silently. :)

Comment (nickvergessen) [inline apps/files/lib/activityhelper.php]: removed

**Comment by LukasReschke** [comment]
:+1:

**Pull request merged**